.github/workflows/rill-cloud.yml

```diff
@@ -2,11 +2,15 @@
 # Each merge to main branch is build with github sha tag and published to Rill Cloud.
 name: Deploy to Rill Cloud
 on:
+  create:
+    branches:
+      - "release**"
```

Comment:
GHA doesnt support filtering on the branches yet. Will need to filter everything for certain branch names only.